Index: Python_algortmos do basico_ao_Avançado/adicoesdelista.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Python_algortmos do basico_ao_Avançado/adicoesdelista.py b/Python_algortmos do basico_ao_Avançado/adicoesdelista.py
new file mode 100644
--- /dev/null	(date 1661524028648)
+++ b/Python_algortmos do basico_ao_Avançado/adicoesdelista.py	(date 1661524028648)
@@ -0,0 +1,8 @@
+lista = ['a', 'b']
+print(lista)
+lista.extend(['c'])
+print(lista)
+lista.append(['d', 'e'])
+print(lista)
+lista.extend(['f','g','h'])
+print(lista)
